static/monsterSort.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	$('select').change(function() {
+		selection = $(this).val();
```

Comment:
"selection" isn't a very useful name.  What type of thing is the value?  Monster-type?